go.mod

```diff
@@ -0,0 +1,10 @@
+module PhilomathesInc/demo-ci-pipelines
```

Comment:
Although you might get away with using this for now, it is not standard, and might break with any future release of go, since this behaviour is not promised.

Reply:
Yes, will fix it. Thanks for pointing out.